src/twisted/web/test/test_http.py

```diff
@@ -1240,6 +1240,34 @@ def test_short(self):
         self.assertEqual(L, [b"a", b"b", b"c", b"1", b"2", b"3", b"4", b"5"])
         self.assertEqual(finished, [b""])
 
+    def test_long(self):
```

Comment:
Very minor suggestion
```suggestion
    def test_fragmented_chunk(self):
```

Reply:
`test_long` is the opposite of `test_short` though. :)

Reply:
Maybe also rename `test_short` to `test_on_shot_chunk`  or something like that.
But really, feel free to ignore.

Reply:
That name doesn't comply with the coding standard. ;)